build.gradle

```diff
@@ -89,12 +89,16 @@ task tagModule() {
     description = "Internal task that creates a tag for the given module"
 }
 
+String currentBranch = System.getenv("BITRISE_GIT_BRANCH")
+
 sonarqube {
     properties {
         property "sonar.projectKey", "bitrise-io_trace-android-sdk"
         property "sonar.organization", "bitrise-io"
         property "sonar.host.url", "https://sonarcloud.io"
-        property "sonar.branch.name", System.getenv("BITRISE_GIT_BRANCH")
-        property "sonar.branch.target", "main"
+        property "sonar.branch.name", currentBranch
```

Comment:
Little bit confused what the implications of this means and ` Feature branches will be targeted on main, main will be not targeted` ? as analyse_code is a manual workflow - does this mean we should only run it on feature branches, or only run it on main? Can you maybe give some more context of what this means?

Reply:
Sure, targeting means the results should contain the issues, that the given branch introduced. So let's say we have a feature branch, where we added 5 lines of code. If we target the check on the main branch, it will check what issues those 5 lines introduced, and will hide the existing issues from the main branch. More information about it can be found [here](https://sonarcloud.io/documentation/branches/overview/).

Reply:
ah thanks for the extra context - could you add that info to the confluence too?

Reply:
Sure, added.